cli/src/test/resources/ide-projects/upgrade-settings/_ide/urls/java/java/17.0.6/linux_x64.sha256

```diff
@@ -0,0 +1 @@
+c2de7dfbd9f8faaa21b4cdd8518f826dd558c9ab24a0616b3ed28437a674a97b
```

Comment:
since when do we add checksums for test projects? Are they actually used? I can not see how this could be used since the `java` installation in our tests is not yet downloaded as archive but only copied as directory tree.
If these `*.sha256` files are all not needed or used, please remove them.